README.md

```diff
@@ -252,7 +279,7 @@ This application is built with the following principles.
             - In-Sequence actions are asynchronous methods that return promises. This is necessary when one asynchronous method depends on the results from a previous asynchronous method. Example: Making an XHR to `command=CreateChatReport` which returns a reportID which is used to call `command=Get&rvl=reportStuff`.
 1. **Actions manage Onyx Data**
     - When data needs to be written to or read from the server, this is done through Actions only.
-    - Public action methods should never return anything (not data or a promise). This is done to ensure that action methods can be called in parallel with no dependency on other methods (see discussion above).
+    - Public action methods should ideally never return anything (not data or a promise). This is done to ensure that action methods can be called in parallel with no dependency on other methods (see discussion above).
```

Comment:
This was inaccurate. Public action methods could potentially never return promises, but in practice they occasionally do because they need to be called by other actions. If you look in `Report.js` and you'll find some examples of this.

Reply:
> because they need to be called by other actions

Gotcha. Do you think it's overkill to say this explicitly?
 ```
 Public action methods should only have return values if they are called by other actions. 
 ```

Reply:
Not at all! I think that would be great to clarify.